### ISSUE TITLE
False negative for unpacked dictionary argument when callable is `Any`

### ISSUE THREAD

**Issue by erictraut** [microsoft/pyright#10061]
Pyright usually reports errors when a `**` operator is used in an argument expression and the unpacked operand is not a `Mapping[str, Any]`. However, in cases where the callable is `Any`, this check is skipped.

```python
def func(cb1: Callable[..., Any], cb2: Any, x: None):
    cb1(**x)  # Error
    cb2(**x)  # Missing error
```

This is related to this mypy issue: https://github.com/python/mypy/issues/18783

**Comment by erictraut** [comment]
This is addressed in pyright 1.1.397.
